etc/picongpu/hemera-hzdr/k80_picongpu.profile.example

```diff
@@ -0,0 +1,85 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ######################################### (edit those lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module purge
+module load gcc/7.3.0
+module load cmake/3.11.3
+module load cuda/10.0
+module load openmpi/2.1.2-cuda100
+module load boost/1.68.0
+
+# Other Software ##############################################################
+#
+module load zlib/1.2.11
+module load c-blosc/1.14.4
+
+module load adios/1.13.1-cuda100
+module load hdf5-parallel/1.8.20-cuda100
+module load libsplash/1.7.0-cuda100
+
+export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:$SPLASH_ROOT:$HDF5_ROOT
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$HDF5_ROOT/lib
+
+module load libpng/1.6.35
+module load pngwriter/0.7.0
+
+# Environment #################################################################
+#
+#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
+
+export PICSRC=$HOME/src/picongpu
```

Comment:
For external users, need to change the path of PICSRC to `export PICSRC=/bigdata/hplsim/external/software/picongpu/workshop`

Reply:
Nope, that was just for the workshop. Just do the following to become the master of your own PIConGPU source on Hemera:

```bash
# after SSH login on Hemera

cd $HOME
mkdir src
cd src/

git clone https://github.com/ComputationalRadiationPhysics/picongpu.git
# this clones PIConGPU 0.4.3 at the moment
# of course you can
#   git checkout ...
# any other version
```